source/JsMaterialX/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jsmaterialx",
```

Comment:
BTW, If we want to publish this, it might be good to do it "sooner". It tried to publish with the name `materialx` but someone already has a package with that name (totally unrelated).